File: PizzeriaService/PizzeriaDataLayer/Models/Order.cs
namespace PizzeriaDataLayer.Models
{
    public class Order
    {
        public int OrderId { get; set; }

        public MenuItem MenuItem { get; set; }

        public int Quantity { get; set; }

        public Customization Customization { get; set; }

        public double Price { get; set; }
    }
}

File: PizzeriaUI/src/app/shared/models/Order.ts
import { Customization } from './customization';
import { MenuItem } from '../../services/menu-item';

export class Order {
    public OrderId!: number;
    public MenuItem!: MenuItem;
    public Quantity!: number;
    public Customization!: Customization;
    public Price!: number;
}
